### DISCUSSION TITLE
Does the plugin handle horizontal overflow in tables with many columns?

### DISCUSSION THREAD

**Discussion by NIANRV97** [awcodes/filament-table-repeater#136] [Q&A]
Does the plugin provide support for managing overflow-x in tables with a large number of columns? If not, what alternatives or configurations could be implemented to prevent layout issues when the content exceeds the container's width?

**Comment by awcodes** [comment]
Actually, it doesn't. Would be a good PR, but can get tricky since it switches to cards below the defined breakpoints.

**Comment by NIANRV97** [comment]
Hi, are you referring to it being complicated because it uses `use Filament\Forms\Components\Select;`? Could you provide an example of why the `Components` would be used?

**Reply by awcodes** [reply]
It is complicated in that the whole table changes its appearance into cards at specific break points. I'm not 100% sure off the top of my head if that would still work correctly with an overflow wrapper. I'm sure it can be done though.

**Comment by andrewdwallo** [comment]
This feature would make this plugin so much better if possible to figure out. I figured out how to make it horizontally scrollable but it does not work well when using Selects in rows.

**Reply by awcodes** [reply]
It does not but, it gets difficult very quickly trying to balance the input view vs the output view. But you can certainly provide an override css for such use cases.
